satellite/metainfo/validation.go

```diff
@@ -260,9 +260,20 @@ func (endpoint *Endpoint) validatePointer(ctx context.Context, pointer *pb.Point
 func (endpoint *Endpoint) validateRedundancy(ctx context.Context, redundancy *pb.RedundancyScheme) (err error) {
 	defer mon.Task()(&ctx)(&err)
 
-	// TODO more validation, use validation from eestream.NewRedundancyStrategy
-	if redundancy.ErasureShareSize <= 0 {
-		return Error.New("erasure share size cannot be less than 0")
+	if endpoint.rsConfig.Validate == true {
+		if redundancy.ErasureShareSize <= 0 ||
+			redundancy.MinReq >= redundancy.RepairThreshold {
```

Comment:
also error if repair threshold > success threshold or if success threshold > max threshold

Reply:
updated with below test cases
// error - ErasureShareSize <= 0
// error - any of the values are negative
// error - MinReq >= RepairThreshold
// error - MinReq >= RepairThreshold
// error - RepairThreshold >= SuccessThreshold
// error - SuccessThreshold >= Total
// ok - valid RS parameters